test/test_core.py

```diff
@@ -5779,6 +5779,15 @@ def test_fs_nodefs_readdir(self):
     self.emcc_args += ['-lnodefs.js']
     self.do_runf('fs/test_nodefs_readdir.c', 'success')
 
+  @requires_node
+  def test_fs_nodefs_statfs(self):
+    # externally setup an existing folder structure: existing/a
+    if self.get_setting('WASMFS'):
+      self.set_setting('FORCE_FILESYSTEM')
+    os.makedirs(os.path.join(self.working_dir, 'existing', 'a'))
```

Comment:
I think the working_dir part here is implicit.. the current working directory will always be the working dir when the test is running.  So I think you can just do `os.makedirs('existing/a')` here.

Reply:
(Oh i see this is just copied from above.. I guess that test can be updated too).